.github/workflows/rust.yml

```diff
@@ -30,6 +30,25 @@ jobs:
   build_and_test:
     name: Rust and clippy tests
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sed:
+          - GNU
+        bash_version:
+          - "5.3-alpha"
```

Comment:
Do we actually want to support alpha Bash versions?

Reply:
Also, when I asked "Do we have a minimum supported version of Bash?", @Ph0enixKM replied "Minimum supported I think would be 3.0." Should probably include earlier versions here.

Reply:
Running tests on alpha version will be helpful, if it is allowed to failure. But, iirc, GitHub Actions does not support allow-failure which exists on Travis. So removing alpha version is an option.

Reply:
Edited! a6e56e12579c42a4328bc816361a9e4593f84160

Reply:
3.1 and 3.0 fail. I've created #621, but if bash array introduced at 3.2, it seems to be hard to support the elder versions.

Reply:
I've removed 3.1 and 3.0 from the tests.

Reply:
Yeah, I think that to support those version we should focus on them but having the tests to run for those version when we are not ready is just annoying.

Reply:
Along with a builtin to return the Bash version (suggested in a comment on a different PR) we should also add an annotation specifying the minimum Bash version to support a given function or `main` block.

Reply:
Let's keep 3.2 as a minimal requirement then! Latest MacOS comes with Bash version 3.2 so it's good enough.